subprojects/docs/src/snippets/java/multiproject/groovy/settings.gradle

```diff
@@ -1,5 +1,3 @@
 rootProject.name = 'multiproject'
 
-// tag::include-projects[]
 include "shared", "api", "services:webservice", "services:shared"
```

Comment:
Are the `"services:webservice", "services:shared"` includes still needed?

Reply:
At the moment the project listing from this one is displayed here: https://docs.gradle.org/nightly/userguide/intro_multi_project_builds.html#sec:identifying_project_structure

It will probably be possible to merge this snippet with another one - this should become more evident after all the "fat" is cut from this one.

Reply:
:+1: we'll probably look at this again once we concluded the "project hierarchy" topic.